modules/upgrading-mtc-1-8-0.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/upgrading-mtc.adoc
+
+:_content-type: PROCEDURE
+[id="migration-upgrading-mtc-18_{context}"]
+= Upgrading to {mtc-full} to 1.8.0
+
+
+.Procedure
+
+* Determine subscription name(s) and current channels to work with for upgrading, by using either issuing:
+
++
+[source,terminal]
+----
+$ oc -n openshift-migration get sub
+----
++
+Which will return an output similar to the following:
+
++
+[source,terminal]
+----
+NAME                                                                         PACKAGE                SOURCE                 CHANNEL
+mtc-operator                                                                 mtc-operator           mtc-operator-catalog   release-v1.7
+redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace   redhat-oadp-operator   mtc-operator-catalog   stable-1.0
+----
++
+Or for a more programmatic approach, issue:
++
+[source,terminal]
+----
+$ oc -n openshift-migration get sub -ojson | jq -r '.items[] | { name: .metadata.name, package: .spec.name, channel: .spec.channel }'
+----
++
+Which will return an output similar to the following:
++
+[source,terminal]
+----
+{
+  "name": "mtc-operator",
+  "package": "mtc-operator",
+  "channel": "release-v1.7"
+}
+{
+  "name": "redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace",
+  "package": "redhat-oadp-operator",
+  "channel": "stable-1.0"
+}
+----
+
+* For each subscription, patch to move from the {mtc-short} 1.7 channel to the newer channel {mtc-short} 1.8:
+[source,terminal]
+----
+$ oc -n openshift-migration patch subscription mtc-operator --type merge --patch '{"spec": {"channel": "release-v1.8"}}'
+----
++
+The subscription is returned as `patched`:
```

Comment:
I'm not sure what is required here to fix it, but the formatting in the rendered preview has a plus `+` in front of "_The subscription..._" text rather than indenting the line to align with the rest of the block under the bullet.

Reply:
fixed - will check build preview